docs/howtos/smoke-testing-app-services.md

```diff
@@ -7,4 +7,15 @@ It can be done manually using substitution scripts, but we also have scripts tha
 
 The `automation/smoke-test-firefox-ios.py` script will clone (or use a local version) of Firefox iOS and
 run tests against the current application-services worktree.  
-Add the `-h` argument to discover all the script exciting options!
+Add the `-h` argument to discover all of the script's exciting options!
+
+## Android Components
+
+The `automation/smoke-test-android-components.py` script will clone (or use a local version) of
+android-components and run a subset of its tests against the current application-services worktree.
+It tries to only run tests that might be relecant to application-services functionality.
```

Comment:
Nit
```suggestion
It tries to only run tests that might be relevant to application-services functionality.
```

Reply:
I think my typos are getting worse the older I get...